airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams.py

```diff
@@ -497,14 +444,6 @@ def _date_ranges(self, stream_state: Mapping[str, Any]) -> Iterator[dict]:
                 "time_range": {"since": since.to_date_string(), "until": until.to_date_string()},
             }
 
-    @backoff_policy
```

Comment:
Not blocking but it seems the new Async class should be responsible for the retry logic instead of communicating to a calling class it should be retried by raising an exception. Not blocking though